internal/internal_worker.go

```diff
@@ -951,6 +951,9 @@ func (aw *AggregatedWorker) Start() error {
 		if err := aw.workflowWorker.Start(); err != nil {
 			return err
 		}
+		if aw.client.eagerDispatcher != nil {
```

Comment:
What was the reasoning behind this change? (not that I disagree with it)

Reply:
Seems inefficient to register workers that had not started yet and force every eager workflow start attempt to try them over and over. Not a huge optimization, but this code path is also for low latency

Reply:
:+1: Just making sure it wasn't fixing something I couldn't see